tokens.xml

```diff
@@ -3130,6 +3199,7 @@ Whenever Glyph Keeper becomes the target of a spell or ability for the first tim
             <reverse-related count="x">Goblin Goliath</reverse-related>
             <reverse-related>Goblin Instigator</reverse-related>
             <reverse-related count="2">Goblin Marshal</reverse-related>
+            <reverse-related attach="attach">Goblin Morningstar</reverse-related>
```

Comment:
Goblin Morningstar does not attach 45% of the time, so there should be another relation to create a goblin without the attach attribute.
Even though this is less likely to happen, it should probably still be the default (i.e. `exclude` the `attach` relation) otherwise players will need to manually detach the equipment from the token if they use the default action.

Reply:
Added